### PR TITLE
Add tests for replymarkup objects

### PR DESCRIPTION
**Pull request by jlmadurga**
Add tests for ReplyMarkup objects:
* ReplyKeyboardMarkup
* ReplyKeyboardHide
* ForceReplay

### CODE REVIEW

File: tests/test_reply_keyboard_markup.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        self.resize_keyboard = True
+        self.one_time_keyboard = True
+        self.selective = True
+
+        self.json_dict = {
+            'keyboard': self.keyboard,
+            'resize_keyboard': self.resize_keyboard,
+            'one_time_keyboard': self.one_time_keyboard,
+            'selective': self.selective,
+
```
Comment (leandrotoledo) [inline tests/test_reply_keyboard_markup.py]: Got one extra line here

File: tests/test_force_replay.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    """This object represents Tests for Telegram ForceReply."""
+
+    def setUp(self):
+        self.force_reply = True
+        self.selective = True
+
+        self.json_dict = {
+            'force_reply': self.force_reply,
+            'selective': self.selective,
+
```
Comment (leandrotoledo) [inline tests/test_force_replay.py]: Got one extra line here

File: tests/test_reply_keyboard_hide.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    """This object represents Tests for Telegram ReplyKeyboardHide."""
+
+    def setUp(self):
+        self.hide_keyboard = True
+        self.selective = True
+
+        self.json_dict = {
+            'hide_keyboard': self.hide_keyboard,
+            'selective': self.selective,
+
```
Comment (leandrotoledo) [inline tests/test_reply_keyboard_hide.py]: Got one extra line here

**Comment by leandrotoledo** [comment]
Just PEP8 minor comments, other than that it does look good to me!
Thanks for your contribution!

File: tests/test_reply_keyboard_hide.py (hunk 3 above)
Comment (jlmadurga) [inline tests/test_reply_keyboard_hide.py]: Ok. `make pep8` didn't complain.

**Comment by leandrotoledo** [comment]
Yay, all good. Merging;

**Pull request merged**